client/README.md

```diff
@@ -0,0 +1,27 @@
+# Client
+
+## Tools
+Install required tools. You need to install node, 
+
+```sh
+$ npm install -g bower gulp karma
```

Comment:
Global install for `karma` is `karma-cli`. `client/package.json` should be ok, as many packages depend on `karma`.

Possible edit:
```sh
$ npm install -g bower gulp karma-cli
```